frontend/src/components/layout/AppNavBar/AppNavBar.tsx

```diff
@@ -161,7 +163,11 @@ function ReportsDropdown() {
       id="reports"
     >
       {keycloak.hasClaim(Claims.REPORTS_SPL) && (
-        <NavDropdown.Item onClick={() => history.push('/reports/spl')}>SPL Report</NavDropdown.Item>
+        <FeatureToggle tenant="MOTI" hide>
+          <NavDropdown.Item onClick={() => history.push('/reports/spl')}>
```

Comment:
I think it wants a test here for the onClick event, because the snapshot does cover the display of the dropdown item on this line.

Reply:
you are right. Will add a test to cover. Before it was partially covered by the if statement